### ISSUE TITLE
Preparation for update process

### ISSUE THREAD

**Issue by SchraderR** [SchraderR/sp-tarkov-client#116]
- [ ] save local information about installed mods
- [ ] add a merge between local and external information

**Comment by FoldZero** [comment]
nice! looking forward to this, even an notification that online version > local installed version would be great

**Comment by SchraderR** [comment]
Yes. This step is a preparation I want to implement before the 0.3.0 update, which will carry the update process. 
But when all steps are done, it should be done pretty fast because i got a working example locally.

Here is the issue for the update process https://github.com/SchraderR/sp-tarkov-client/issues/107

**Comment by SchraderR** [comment]
Will be summarized in this discussion and ported back https://github.com/SchraderR/sp-tarkov-client/discussions/130
